src/modules/ods/ods.controller.test.ts

```diff
@@ -0,0 +1,31 @@
+import { CUSTOMERS } from '@ukef/constants';
+import { when } from 'jest-when';
+
+import { OdsController } from './ods.controller';
+import { OdsService } from './ods.service';
+
+describe('OdsController', () => {
```

Comment:
I wanted a test to check the validation of the dto (e.g. valid when a urn matching the regex is passed through) but that would require setting up a validationPipe for this test which no other test in the codebase has done. For now I'm going to leave out to stay consistent with the other test suites but one to look at across the board

Reply:
At the moment DTO validations are tested as part of `api/e2e` tests (`root/tests`). Most likely these tests are not part of github pipelines yet.

Would be nice to create `api/e2e` test for `osd`.

Reply:
Yep will add this in a new PR straight after this one (don't have a tonne of time today/tomorrow but keen to get my feature branch tested in dev so have split up the work)